src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -448,4 +472,81 @@ end
 
 end probability_measure
 
+section convergence_implies_limsup_closed_le
+
+open measure_theory set filter bounded_continuous_function
+open_locale topological_space ennreal nnreal bounded_continuous_function
+
+variables [topological_space α] [opens_measurable_space α]
```

Comment:
Since this `variables` declaration corresponds only to one lemma and forces you to redefine alpha in two lemmas, it could be better to not have it.